packages/amplify_datastore/pubspec.yaml

```diff
@@ -4,13 +4,14 @@ version: 0.2.0-nullsafety.0
 homepage: https://github.com/aws-amplify/amplify-flutter/tree/master/packages/amplify_datastore
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
   flutter: ">=1.17.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  amplify_datastore_plugin_interface: 0.2.0-nullsafety.0
+  amplify_datastore_plugin_interface:
+    path: ../amplify_datastore_plugin_interface
```

Comment:
Seems like we should leave this as is?

Reply:
ah thanks again another testing artifact I forgot to remove